core/src/xyz/magicjourney/nebulaquest/dice/Dice.java

```diff
@@ -38,8 +48,13 @@ public void act(float delta) {
     this.viabilityTimer.act(delta);
   }
 
+  protected int debugRoll() {
+    return debugRolls[Math.max(0, Math.min(this.debugRolls.length - 1, this.rollNumber++))];
+  }
+
```

Comment:
Gdzie tego używamy? Czy służy do dev testów?

Reply:
Tak, służy do testowania. Dokładniej to ustawia rzuty kostką według tablicy debugRolls. Włączasz to w metodzie roll: 
```
public void roll(Consumer<Integer> callback) {
    //int result = this.random.nextInt(11) + 1; 
    int result = this.debugRoll(); 

    if (this.isRolling) {
      return;
    }
```

W konstruktorze masz dokładną konfigurację: 
```
 this.debugRolls = new int[] {
      5, 5, 5,
      5, 2, 
      5, 6,
      5, 12
    };
```